src/components/Select.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+export const SelectWrapper = styled.select`
+    
+   height: 50px;
+   width: 535px;
+   border: 1px solid #E8E8E8;
+   border-radius: 6px;
+`;
+
+export default function Select(){
```

Comment:
seria uma boa tirar o default pra facilitar a identificação ao importar, e transformar em arrow function por questões de performance
```suggestion
export const Select = () => {
```

Reply:
Ja arrumei essa parte